src/components/projectSelect/ProjectOptionGroup.tsx

```diff
@@ -37,9 +39,18 @@ export const ProjectOptionGroup: React.FC<OptionGroupProps> = ({
   name,
   projects,
   onClick,
+  repoIdentifier,
+  isProjectSettingsPage = false,
 }) => (
   <OptionGroupContainer>
-    <Overline>{name}</Overline>
+    {repoIdentifier === "" || !isProjectSettingsPage ? (
+      <Overline>{name} </Overline>
+    ) : (
+      <OverlineHover>
+        <Overline onClick={() => onClick(repoIdentifier)}>{name} </Overline>
```

Comment:
Sorry I'm not sure if this is the right place to actually comment this but I try 😅  I love the look/feel of the clickable header! Testing locally though, it doesn't look like it's directing to the right place. Here if I click "evergreen-ci/evergreen" it redirects to the page for evergreen smoke test.
![image](https://user-images.githubusercontent.com/26798134/156645030-5f8593db-ec83-4944-81e8-0ac240b11012.png)